### ISSUE TITLE
Is guides.rubyonrails.org serving 7.1 docs for 7.0.4.3 release?

### ISSUE THREAD

**Issue by cornu-ammonis** [rails/rails#47666]
Hi I noticed that with the `7.0.4.3` release today, the Rails caching guide has changed to say that connection pooling is on by default: https://guides.rubyonrails.org/caching_with_rails.html#connection-pool-options says 
> By default, [:mem_cache_store](https://guides.rubyonrails.org/caching_with_rails.html#activesupport-cache-memcachestore) and [:redis_cache_store](https://guides.rubyonrails.org/caching_with_rails.html#activesupport-cache-rediscachestore) are configured to use connection pooling.

However looking at the release diff, none of the caching code changes from `7.0.4.2` -> `7.0.4.3`. The code to actually do this is _not_ in `7.0.4.3` which can also be confirmed by looking at it directly based on the tag like this: https://github.com/rails/rails/blob/v7.0.4.3/activesupport/lib/active_support/cache.rb which does not include changes from the PR to make connection pooling on by default https://github.com/rails/rails/pull/45235 - looking at that PR it appears to be slated for release with `7.1`.

And indeed if I look at the docs sourcecode for `7.0.4.3` it says connection pooling is off which, again, is not what the guide page is currently serving:
https://github.com/rails/rails/blob/v7.0.4.3/guides/source/caching_with_rails.md#connection-pool-options

And further, I can see a version mismatch on [this page](https://guides.rubyonrails.org/index.html) which says 7.1 and 7.0.4.3 in different places:

![Screenshot 2023-03-13 at 4 07 44 PM](https://user-images.githubusercontent.com/20233388/224852255-56636338-ca6b-4033-acca-7e48b83bfc46.png)

I am having trouble explaining this discrepancy and I am concerned something may be wrong with the docs or the release.

**Comment by zzak** [comment]
7.0.4.3 was a security release so it should only have contained the necessary patches for the CVEs.

That said, this looks weird:

> These are the new guides for Rails 7.1 based on [v7.0.4.3](https://github.com/rails/rails/tree/v7.0.4.3).

I can look into this in a bit 🤔

**Comment by cornu-ammonis** [comment]
Possibly related, I see a 7.1 alpha tag when visiting this other link https://api.rubyonrails.org/classes/ActiveSupport/Cache/MemCacheStore.html
![Screen Shot 2023-03-13 at 4 22 45 PM](https://user-images.githubusercontent.com/20233388/224853819-08da87fc-3c59-4d18-a826-3dcc9e419027.png)

**Comment by cornu-ammonis** [comment]
Looks like this is resolved. Curious what happened, but closing 😄

**Comment by zzak** [comment]
@cornu-ammonis Yeah, sorry for the delay. The server that hosts `[api,guides,contributors].ror.org` was having issues yesterday and after fixing them I haven't tried to regenerate the last 7.0.4.3 release -- it's probably ok for now?
